robottelo/cli/base.py

```diff
@@ -275,7 +275,8 @@ def _get_username_password(cls, username=None, password=None):
 
     @classmethod
     def execute(cls, command, user=None, password=None, output_format=None,
-                timeout=None, ignore_stderr=None, return_raw_response=None):
+                timeout=120, ignore_stderr=None, return_raw_response=None,
+                conn_timeout=10):
```

Comment:
As these two timeouts are already confusing, i'd suggest to name this one `connection_timeout` to make it absolutely clear.